src/iosource/Packet.h

```diff
@@ -14,10 +15,14 @@ typedef struct bpf_timeval pkt_timeval;
 typedef struct timeval pkt_timeval;
 #endif
 
+#include "pcap.h" // For DLT_ constants
+#include "NetVar.h" // For BifEnum::Tunnel
```

Comment:
Use `zeek/` prefix for new includes in headers.

Reply:
Fixed here and in one other place too.